app/views/organizations/_organization_card_layout.html.erb

```diff
@@ -13,8 +13,10 @@
               </summary>
 
               <details-menu class="dropdown-menu dropdown-menu-sw">
-                <%= link_to new_assignment_organization_path(organization), class: 'dropdown-item btn-link btn-block text-left', role: 'button' do %>
-                  Create assignment
+                <% unless organization.archived? %>
+                  <%= link_to new_assignment_organization_path(organization), class: 'dropdown-item btn-link btn-block text-left', role: 'button' do %>
```

Comment:
Why did we decide to remove the "Create assignment" button here rather than disabling it, like in the `organizations/show` view?

Reply:
I left it in the classroom#show page because it's a page people are used to dealing with (and seeing the button there) and it seemed standard to leave the button in place but disable it if it can't be used.

I didn't include it in the dropdown because that's a new menu and feels more like, "What actions can I take on this classroom right now". It also seems more complicated to indicate that an item in a dropdown menu can't be selected vs making a button inactive.

I've pinged @femmebot to weigh in as it's definitely not consistent and I don't want to cause more confusion in my attempts to reduce confusion.

Reply:
I confirmed with @femmebot that this behavior is consistent with our designs elsewhere.

For the button, we want to disable it.

For the dropdown, we only want to include actions the user can take.